src/jrd/RecordSourceNodes.h

```diff
@@ -1081,6 +1081,24 @@ class UnlistFunctionSourceNode : public TableValueFunctionSourceNode
 	}
 };
 
+class GenSeriesFunctionSourceNode : public TableValueFunctionSourceNode
+{
+public:
+	explicit GenSeriesFunctionSourceNode(MemoryPool& pool) : TableValueFunctionSourceNode(pool)
```

Comment:
It's not the common convention to put `: ...` in the same line as the constructor.